test/source/TestString.inl

```diff
@@ -1163,7 +1163,7 @@ int TEST_STRING_NAME()
 		typename StringType::value_type buf[64];
 
 		StringType str(LITERAL("abcdefghijklmnopqrstuvwxyz"));
-		str.copy(buf, 10, sizeof buf);
+		str.copy(buf, 10, 0);
```

Comment:
Yeah.  I fixed in another branch.  Will merge soon.

On Sun, 27 Mar 2016, 6:05 a.m. DragoonX6, <notifications@github.com> wrote:

> In test/source/TestString.inl
> <https://github.com/electronicarts/EASTL/pull/37#discussion_r57528059>:
>
> > @@ -1163,7 +1163,7 @@ int TEST_STRING_NAME()
> >  		typename StringType::value_type buf[64];
> >
> >  		StringType str(LITERAL("abcdefghijklmnopqrstuvwxyz"));
> > -		str.copy(buf, 10, sizeof buf);
> > +		str.copy(buf, 10, 0);
>
> Pretty sure that's still wrong, as the verify statement has a 9 character
> literal but compares 10 characters.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/electronicarts/EASTL/pull/37/files/dd462a4d37dc737fcfa5563b74e3b4df56d3660d#r57528059>
>